app/models/customer.rb

```diff
@@ -33,7 +33,7 @@ def shows ; self.showdates.map(&:show).uniq ; end
   
   validates_format_of :email, :if => :self_created?, :with => /\A\S+@\S+\z/
 
-  EMAIL_UNIQUENESS_ERROR_MESSAGE = 'has already been registered.'
+  EMAIL_UNIQUENESS_ERROR_MESSAGE = 'has already been taken'
```

Comment:
Why is it being changed?

Reply:
Oh yeah, there were so many errors with that I forgot I could change this back. Good catch.